packages/@aws-cdk/aws-rds/lib/database-secret.ts

```diff
@@ -33,6 +34,18 @@ export interface DatabaseSecretProps {
    * @default " %+~`#$&*()|[]{}:;<>?!'/@\"\\"
    */
   readonly excludeCharacters?: string;
+
+  /**
+   * Whether to replace this secret when the password changes.
+   *
+   * This is achieved by overriding the logical id of the AWS::SecretsManager::Secret
+   * with a hash of the options that influence the password generation. This
+   * way a new secret will be created when the password is regenerated and the
+   * cluster or instance consuming this secret will have its credentials updated.
+   *
+   * @default false
+   */
+  readonly replaceOnPasswordChanges?: boolean;
```

Comment:
Should this be more appropriately named `replaceOnPasswordCriteriaChanges?` or something like that? I'm afraid the name and documentation as-is may mislead people to think the secret may be replaced when rotation is configured.

Reply:
I'm ok with this name, it's indeed better. @skinny85 do you validate this name?

Reply:
Yes 🙂